# this variable holds who is assigned for the task
string assignedTo
string humanTrustsRobot
string humanType
string humanMood
string robotType
uint32 task_cnt
---
bool success
